itou/www/dashboard/views.py

```diff
@@ -141,15 +141,35 @@ def edit_user_info(request, template_name="dashboard/edit_user_info.html"):
     return render(request, template_name, context)
 
 
+def is_user_from_sender_prescriber_organization(user, job_application):
+    return (
+        user.is_prescriber
+        and job_application.is_sent_by_authorized_prescriber
+        and job_application.sender_prescriber_organization_id
+        in user.prescribermembership_set.values_list("organization_id", flat=True)
+    )
```

Comment:
J'aurais bien vu cette logique quelque part côté modèle plutôt que dans la vue.

On peut envisager ça :

```python
(user
    .prescriberorganization_set
    .filter(
        pk=job_application.sender_prescriber_organization_id,
        is_authorized=True,
        prescribermembership__is_active=True,
    )
    .exists())
```

Un peu plus cher en jointure SQL, mais ça permettrai de déplacer cette logique.

À débattre.

Reply:
Pas sûr que ce soit plus cher donc why not.
Après je ne pense pas que cela aie sa place dans le modèle car cela prend `job_application.sender_prescriber_organization_id` en argument.

Reply:
Je me dis que si on la pense comme ça, on peut imaginer que ça ait une utilité ailleurs :

```django
user.is_member_of_authorized_prescriber_organization(org_id)
```

Reply:
Vendu.